dropwizard-testing/src/test/java/io/dropwizard/testing/app/PersonResourceTest.java

```diff
@@ -21,6 +24,14 @@
  * Tests {@link ResourceTestRule}.
  */
 public class PersonResourceTest {
+    @Provider
+    public class DummyExceptionMapper implements ExceptionMapper<WebApplicationException> {
```

Comment:
The class should be static, because it doesn't access any inner state of `PersonResourceTest`.
Also I beleive we could relax the class access level from public to private, because it's accessed only in `PersonResourceTest`.